dali/pipeline/operators/reader/parser/coco_parser.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_READER_PARSER_COCO_PARSER_H_
+#define DALI_PIPELINE_OPERATORS_READER_PARSER_COCO_PARSER_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "dali/pipeline/operators/reader/parser/parser.h"
+
+namespace dali {
+
+struct Annotation {
```

Comment:
As a warning, COCO annotations are not just bbox and label. They can include keypoints & segmentation masks too.